seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -273,6 +273,38 @@ public void testOverviewFilterByTag() {
                         });
     }
 
+    @Test
+    public void testUpdateTags() {
```

Comment:
Please add a test case for update tag failed.

Reply:
done